client/core/notification.go

```diff
@@ -148,6 +166,38 @@ type OrderNote struct {
 	Order *Order `json:"order"`
 }
 
+const (
+	OrderLoadFailureSubject     = "Order load failure"
+	OrderPlacedSubject          = "Order placed"
+	MissingMatchesSubject       = "Missing matches"
+	WalletMissingSubject        = "Wallet missing"
+	MatchStatusErrorSubject     = "Match status error"
+	MatchRecoveryErrorSubject   = "Match recovery error"
+	NoFundingCoinsSubject       = "No funding coins"
+	OrderCoinErrorSubject       = "Order coin error"
+	PreimageSentSubject         = "preimage sent"
+	CancelPreimageSentSubject   = "cancel preimage sent"
+	MissedCancelSubject         = "Missed cancel"
+	OrderBookedSubject          = "Order booked"
+	NoMatchSubject              = "No match"
+	OrderCanceledSubject        = "Order canceled"
+	CancelSubject               = "cancel"
+	MatchesMadeSubject          = "Matches made"
+	SwapErrorSubject            = "Swap error"
+	SwapsInitiatedSubject       = "Swaps initiated"
+	RedemptionErrorSubject      = "Redemption error"
+	MatchCompleteSubject        = "Match complete"
+	RefundFailureSubject        = "Refund Failure"
+	MatchesRefundedSubject      = "Matches Refunded"
+	MatchRevokedSubject         = "Match revoked"
+	RevokeSubject               = "revoke"
+	MatchRecoveredSubject       = "Match recovered"
+	CancellingOrderSubject      = "Cancelling order"
+	OrderStatusUpdateSubject    = "Order status update"
+	MatchResolutionErrorSubject = "Match resolution error"
+	FailedCancelSubject         = "Failed cancel"
+)
+
```

Comment:
I'd prefer "Subject" be the prefix instead.

Reply:
Second that.

Reply:
Done